test/upgradeableProxy/DispatcherRC4.upgrade.t.sol

```diff
@@ -279,8 +260,8 @@ contract DispatcherRC4MidwayUpgradeTest is ChannelHandShakeUpgradeUtil, UpgradeT
         assertEq(2, nextSequenceSendValue);
 
         // Now recv uch packet
-
         bytes memory appData = abi.encodePacked("hello using mw stack");
+        uint256 mwBitmap = 1;
```

Comment:
what does "1" mean here? is it mw1.MW_ID()?

Reply:
yes, it should be the MW_ID (since there's only one MW in the stack)

I'm not sure if we should get rid of the MW_ID stored in the UCH (and also if we should remove how it is currently encoded in the universal packet data [here](https://github.com/open-ibc/vibc-core-smart-contracts/blob/d804d0d47cb4b1c4af353b959dbdc7baa9d48a04/contracts/core/UniversalChannelHandler.sol#L97)) - it seems like we don't need it anymore, but it might break dapps who are expecting this MW_ID to be encoded in the previous format with MW_ID included?

Reply:
Should we use "mw1.MW_ID()" here instead of "1" then? This is just a nit so I've approved the PR.

Reply:
sure, changed it to be `uch.MW_ID`